src/lib/ansi/ansi-palette-analyser.js

```diff
@@ -34,8 +46,14 @@ export function analyzePalette(palette) {
   };
 }
 
+/**
+ *
+ * @param {boolean} isDarkPalette
+ * @return {Array<ColorsType>}
+ */
 function getColorsToTest(isDarkPalette) {
-  const result = [...colors, isDarkPalette ? 'white' : 'black'];
+  const result = [...COLORS, isDarkPalette ? 'white' : 'black'];
 
+  // @ts-ignore
```

Comment:
**note:** I have the same case within the `cc-ansi-palette` and I haven't found a solution for this either, I'll go with a `ts-expect-error` / ignore as well